src/components/AccountDropdown/AccountChildOption.tsx

```diff
@@ -9,25 +9,23 @@ type AccountChildRowProps = {
 } & MenuItemOptionProps
 
 export const AccountChildOption = forwardRef<AccountChildRowProps, 'button'>(
-  ({ title, cryptoBalance, symbol, children, ...props }, ref) => {
-    const Component = (
-      <MenuItemOption ref={ref} {...props}>
-        <Stack direction='row' justifyContent='space-between' fontSize='sm' spacing={4}>
-          <RawText fontWeight='bold' whiteSpace='nowrap'>
-            {title}
-          </RawText>
-          <Amount.Crypto
-            whiteSpace='nowrap'
-            color='gray.500'
-            fontWeight='medium'
-            value={cryptoBalance}
-            symbol={symbol}
-          />
-        </Stack>
-        {children}
-      </MenuItemOption>
-    )
-    return Component
-  },
+  ({ title, cryptoBalance, symbol, children, ...props }, ref) => (
```

Comment:
Just a refactor to return in-line. No code touched here.

Reply:
That's also correct semantically now since the previous `Component` wasn't a component but rather an `Element` / JSX node, nice catch!